objective.py

```diff
@@ -26,29 +29,79 @@ def set_data(self, X, y):
         self.X, self.y = X, y
         self.n_samples, self.n_features = X.shape
 
+        # the following code fits the path once using the same path-stopping
+        # criteria that glmnet uses to find a reasonable lambda path in order
+        # to obtain a reasonable lambda sequence
         if self.lambda_min_ratio is None:
             self.lambda_min_ratio = 1e-2 if self.n_samples < self.n_features else 1e-4
 
         lambda_max = self._get_lambda_max()
-        self.lambdas = np.logspace(
+        lambdas = np.logspace(
```

Comment:
I think it's clearer to use np.geomspace and it should be equivalent, right ?

Reply:
You're right, thanks, I thought geomspace would be with the 10 base too because logspace is.